fms_sdg/generate_data.py

```diff
@@ -19,19 +19,14 @@
 
 
 def generate_data(
-    # model_family = (model_family,)
-    # model_name = (model,)
-    num_outputs_to_generate: int,
-    num_prompt_instructions: int,
     max_gen_requests: int,
     data_path: str,
     output_dir: str,
-    lm_cache: str,
-    include_data_path: str,
-    include_builder_path: str,
-    prompt_file_path: str,
-    console_output: bool = True,
-    restart_generation: bool = False,
+    task_kwargs: Dict,
+    builder_kwargs: Dict,
+    include_data_path: Optional[str] = None,
+    include_builder_path: Optional[str] = None,
+    restart_generation: Optional[bool] = False,
```

Comment:
As we have a valid default why not?
```suggestion
    restart_generation: bool = False,
```

Reply:
Ah yes of course